specification/iotoperations/IoTOperations.Management/models/dataflows/dataflowEndpoints.tsp

```diff
@@ -332,6 +374,30 @@ model DataflowEndpointMqtt {
   cloudEventMapping?: CloudEventMappingType;
 }
 
+@doc("Mqtt endpoint Authentication properties. NOTE - only authentication property is allowed per entry.")
+model DataflowEndpointMqttAuthentication {
+  @doc("Mode of Authentication.")
+  method: MqttAuthMethod;
+
+  ...ManagedIdentity;
+
+  @doc("Kubernetes service account token authentication. Default audience if not set is aio-mq-internal")
+  serviceAccountTokenSettings?: DataflowEndpointAuthenticationServiceAccountToken;
+
+  @doc("X.509 certificate authentication.")
+  x509Settings?: DataflowEndpointAuthenticationX509;
```

Comment:
I believe we agreed this should be `x509CertificateSettings`

https://msazure.visualstudio.com/One/_workitems/edit/28771593